test/standard/_expected/async_css/asset_tag.html

```diff
@@ -0,0 +1,12 @@
+<script>/*!
```

Comment:
@kogakure Here you can see the output that you'll get if you set your site configuration to:

```
asset_pipeline:
  async_css: true
```